packages/react-native-editor/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ For each user feature we should also add a importance categorization label  to i
 
 ## Unreleased
 -   [***] Fix issue when backspacing in an empty Paragraph block [#56496]
+-   [**] Add OfflineStatus component [#56627]
```

Comment:
Given we use this change log to produce user-facing release notes, I might suggest focusing on the impact rather than the implementation details. 
```suggestion
-   [**] Editor displays user of network connection status when offline [#56627]
```